2025/lpc/future-of-platform-security-measurement.md

```diff
@@ -0,0 +1,573 @@
+  + Supervisor Mode Access Prevention (SMAP), memory encryption
+* **HSI-5:** Out-of-band attestation (planned, not yet implemented)
+
+<!--
+Notice the progression from basic to advanced security features. Each level
+adds more sophisticated protections.
+-->
+
+---
+
+# Intel Boot Guard
+
+* A processor feature
+* Prevents the machine from running firmware images not released (signed) by
+  the system vendor
+* It forms a Root of Trust (RoT) by fusing in cryptographic keys into the
+  processor itself
+
+<!--
+
+TBD: Extend  or drop?
+
+-->
+
+---
+layout: two-cols-header
+class: text-center
+---
+
+# Intel Boot Guard and Management Engine
+
+Intel Boot Guard status reporting with disabled ME
+
+* open-source firmware users typically prefers the ME to stay disabled
+* If IBG is configured correctly, firmware is still verified on boot
+* fwupd is lacking an interface to confirm this correct configuration
+
+::left::
+
+<figure>
+  <img src="/@fs/repo/img/hsi_me_enabled.png" width="200">
+  <figcaption>ME enabled</figcaption>
+</figure>
+
+::right::
+
+<figure>
+  <img src="/@fs/repo/img/hsi_me_disabled.png" width="200">
+  <figcaption>ME disabled</figcaption>
+</figure>
+
+<!--
+
+The HSI score is radically different.
+
+In reality, the security posture is the same - or one could argue the one with
+ME disabled is even better due to reduced surface.
+
+References:
+- https://github.com/Dasharo/dasharo-issues/issues/463
+- https://github.com/fwupd/fwupd/issues/6011
+
+-->
+
+---
+
+# Intel Boot Guard and Management Engine
+
+There are more HSI checks reading from ME PCI space
+
+| HSI check | ME >=11 | ME >= 18 |
+| :-- | :-- | :-- |
+| <span style="color: red;">ME not in manufacturing mode</span> | <span style="color: red;">HFSTS1</span> | <span style="color: red;">HFSTS1</span> |
+| <span style="color: green;">ME Flash Descriptor Override</span> | <span style="color: green;">HFSTS1</span> | <span style="color: green;">HFSTS1</span> |
+| <span style="color: green;">Intel BootGuard: Enabled</span> | <span style="color: green;">HFSTS6</span> | <span style="color: green;">HFSTS5</span> |
+| <span style="color: green;">Intel BootGuard: Verified</span> | <span style="color: green;">HFSTS6</span> | <span style="color: green;">N/A</span> |
+| <span style="color: green;">Intel BootGuard: ACM</span> | <span style="color: green;">HFSTS6</span> | <span style="color: green;">HFSTS5</span> |
+| <span style="color: green;">Intel BootGuard: Policy</span> | <span style="color: green;">HFSTS6</span> | <span style="color: green;">N/A</span> |
+| <span style="color: red;">Intel BootGuard: OTP fuse</span> | <span style="color: red;">HFSTS6</span> | <span style="color: red;">HFSTS6</span> |
+
+---
+
+<!--
+
+TBD: Only two of them are failing with ME disabled? Why? Can we really read out
+some of the registers, but not all of them, even when ME is disabled?
+
+-->
+
+# Intel Boot Guard and Management Engine
+
+Alternative ways of checking the failing checks
```

Comment:
@miczyg1 @mkopec Do we have any suggestions on how one could check those failing tests in another way, when ME was disabled?

I recall there was some short discussion in one of the previous DUGs during Richard's presentation.